### DISCUSSION TITLE
Specify column widths?

### DISCUSSION THREAD

**Discussion by BarbaraPruz** [table-library/react-table-library#129] [Q&A] [answered]
Just getting started and so far, this looks like the right package for our app.
But one styling issue that I can seem to figure out. This seems like a really common requirement. I want to size some of my columns. For example, I have a "State" column with values that will only be 2 characters and so I want to make that column smaller than others. Likewise, I have an email column that I would like to have a minimum width.
Is there a way to do this?

**Comment by rwieruch** [comment, marked as answer]
You can layout each column individually. Does this help? https://react-table-library.com/?path=/docs/features-layout--custom

**Reply by BarbaraPruz** [reply]
Success! I was missing the `layout={{ custom: true }}`
Thanks for quick reply.
